scripts/generate_zips.ts

```diff
@@ -3,10 +3,21 @@ import * as path from "path";
 import * as util from "util";
 import * as childProcess from "child_process";
 
+interface TemplateData {
+	filepath: string;
+	list: TemplateList;
+}
+
+interface TemplateList {
+	templates: {[name: string]: string};
+}
+
 const exec = util.promisify(childProcess.exec);
 
 const outputDir = path.join(__dirname, "..", "dist");
 const templateDir = path.join(__dirname, "..", "templates");
+const templateListJsonDir = path.join(__dirname, "..", "public");
+const templateListJsonBaseUrl = "https://github.com/akashic-contents/templates/releases/latest/download/";
```

Comment:
https://docs.github.com/ja/repositories/releasing-projects-on-github/linking-to-releases